tools/instrument_flag_consistency_check.php

```diff
@@ -0,0 +1,130 @@
+<?php
+    function pad_or_trunc ($str, $len) {
+        if (strlen($str) > $len) {
+            return substr($str, 0, $len-1) . ".";
+        } else if (strlen($str) == $len) {
+            return $str;
+        } else {
+            return str_pad($str, $len);
+        }
+    }
+    
+    require_once __DIR__ . "/../vendor/autoload.php";
+    $client = new NDB_Client();
+    $client->makeCommandLine();
+    $client->initialize(__DIR__ . "/../project/config.xml");
+    $DB = Database::singleton();
+    

```

Comment:
Which tools are using the `generic_includes.php` file? I looked over a few of them and saw them doing exactly this.